src/Microsoft.Azure.WebJobs.Host/Hosting/WebJobsHostBuilderExtensions.cs

```diff
@@ -28,6 +29,13 @@ public static IHostBuilder ConfigureWebJobsHost(this IHostBuilder builder, Actio
                 config.AddEnvironmentVariables();
             });
 
+
+            builder.ConfigureLogging((context, logging) =>
+            {
+                // Loggers can be configured under "Logging" in config
+                logging.AddConfiguration(context.Configuration.GetSection("Logging"));
+            });
```

Comment:
@fabiocav -- I decided to auto-register this. It seems like a common pattern and is overrideable if needed (I have a test that verifies this).

Reply:
Sounds good.  Do we need to think about whether this  would be on the root, though? In the case of host.json, this would be scoped under "AzureFunctionsJobHost", do we need to special case it?

Reply:
I guess we could also have some logic in Script where `AddConfiguration` is called with the appropriate section.

Reply:
Yeah, you bring up a good point. I'll actually pull this out and we can chat about it. The logging docs show that you should add this yourself -- but I was trying to eliminate some concepts for people getting started. But maybe it won't work quite as smoothly as I'd hoped. It's easy to add back later.